### DISCUSSION TITLE
Flatmaps - NMT space

### DISCUSSION THREAD

**Discussion by SjoerdMurris** [PRIME-RE/prime-re.github.io#82] [General]
Hi everyone,

I've been looking to plot flatmaps in monkey template space (NMT). But perhaps someone here has already done so? If that's the case, it would be great if someone could share or provide additional info on that. :-)

I have made flatmaps of D99 in the (somewhat distant) past before with freesurfer, and flatmaps of individual monkeys.
So I can (and will) of course go ahead and share the maps here, once I've made them. But again, do reach out if you've gone through this process already!

**Comment by PRIME-DRE** [comment]
Hi!

We used NMT v2 template and ROIs to map to surface reconstructions (not 
flatmaps) in this paper: 
https://doi.org/10.1016/j.neuroimage.2021.118283 
<https://doi.org/10.1016/j.neuroimage.2021.118283>

Please see Methods and e.g. Figures 4 and 5. The problem is that surface 
reconstruction and visualization was done in BrainVoyager (commercial 
package), although the same functionality now exists in NeuroElf package 
that runs in MATLAB.

I can share intermediate and final files if you think it might be useful.

Cheers,
Igor

-- 
Dr. Igor Kagan
Head of Decision and Awareness Group, German Primate Center (DPZ)
Kellnerweg 4, Goettingen 37077, Germany
http://www.dpz.eu/dag



On 12/4/2023 1:57 PM, 'SjoerdMurris' via NHP-MRI wrote:
>
> Hi everyone,
>
> I've been looking to plot flatmaps in monkey template space (NMT). But 
> perhaps someone here has already done so? If that's the case, it would 
> be great if someone could share or provide additional info on that. :-)
>
> I have made flatmaps of D99 in the (somewhat distant) past before with 
> freesurfer, and flatmaps of individual monkeys.
> So I can (and will) of course go ahead and share the maps here, once 
> I've made them. But again, do reach out if you've gone through this 
> process already!
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/PRIME-RE/prime-re.github.io/discussions/82>, or 
> unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/BBSQQAIIKRKSLJNPM3T7TODYHXCCZAVCNFSM6AAAAABAF54T6WVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEZDENRZGA>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>
> -- 
> You received this message because you are subscribed to the Google 
> Groups "NHP-MRI" group.
> To unsubscribe from this group and stop receiving emails from it, send 
> an email to ***@***.***
> To view this discussion on the web visit 
> https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/82%40github.com 
> <https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/82%40github.com?utm_medium=email&utm_source=footer>.

--------------u4ST6ARkbHSedjefODSYnNVb
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: 8bit

<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  </head>
  <body text="#000000" bgcolor="#FFFFFF">
    <p>Hi!</p>
    <p>We used NMT v2 template and ROIs to map to surface
      reconstructions (not flatmaps) in this paper: <a
        class="anchor doi anchor-default"
        href="https://doi.org/10.1016/j.neuroimage.2021.118283"
        target="_blank" rel="noreferrer noopener"
        aria-label="Persistent link using digital object identifier"
        title="Persistent link using digital object identifier"
style="box-sizing: border-box; margin: 0px; padding: 0px; background-color: transparent; word-break: break-word; color: rgb(31, 31, 31); transition: color 0.3s ease 0s, border-bottom-color 0.3s ease 0s; text-decoration: none; display: inline-block; outline-width: 0px; font-family: ElsevierSans, Arial, Helvetica, Roboto, &quot;Lucida Sans Unicode&quot;, &quot;Microsoft Sans Serif&quot;, &quot;Segoe UI Symbol&quot;, STIXGeneral, &quot;Cambria Math&quot;, &quot;Arial Unicode MS&quot;, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal;"><span
          class="anchor-text"
style="box-sizing: border-box; margin: 0px; padding: 0px; border-bottom: 2px solid rgb(235, 101, 0); transition: border-bottom-color 0.3s ease 0s;">https://doi.org/10.1016/j.neuroimage.2021.118283</span></a></p>
    <p>Please see Methods and e.g. Figures 4 and 5. The problem is that
      surface reconstruction and visualization was done in BrainVoyager
      (commercial package), although the same functionality now exists
      in NeuroElf package that runs in MATLAB. <br>
    </p>
    <p>I can share intermediate and final files if you think it might be
      useful.</p>
    <p>Cheers,<br>
      Igor</p>
    <pre class="moz-signature" cols="72">-- 
Dr. Igor Kagan
Head of Decision and Awareness Group, German Primate Center (DPZ)
Kellnerweg 4, Goettingen 37077, Germany
<a class="moz-txt-link-freetext" href="http://www.dpz.eu/dag">http://www.dpz.eu/dag</a></pre>
    <p><br>
    </p>
    <p><br>
    </p>
    <p></p>
    <div class="moz-cite-prefix">On 12/4/2023 1:57 PM, 'SjoerdMurris'
      via NHP-MRI wrote:<br>
    </div>
    <blockquote type="cite"
***@***.***">
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <p dir="auto">Hi everyone,</p>
      <p dir="auto">I've been looking to plot flatmaps in monkey
        template space (NMT). But perhaps someone here has already done
        so? If that's the case, it would be great if someone could share
        or provide additional info on that. :-)</p>
      <p dir="auto">I have made flatmaps of D99 in the (somewhat
        distant) past before with freesurfer, and flatmaps of individual
        monkeys.<br>
        So I can (and will) of course go ahead and share the maps here,
        once I've made them. But again, do reach out if you've gone
        through this process already!</p>
      <p
style="font-size:small;-webkit-text-size-adjust:none;color:#666;">—<br>
        Reply to this email directly, <a
href="https://github.com/PRIME-RE/prime-re.github.io/discussions/82"
          moz-do-not-send="true">view it on GitHub</a>, or <a
href="https://github.com/notifications/unsubscribe-auth/BBSQQAIIKRKSLJNPM3T7TODYHXCCZAVCNFSM6AAAAABAF54T6WVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEZDENRZGA"
          moz-do-not-send="true">unsubscribe</a>.<br>
        You are receiving this because you are subscribed to this
        thread.<img
src="https://github.com/notifications/beacon/BBSQQAKPUIIDRNEC7J2IFPTYHXCCZA5CNFSM6AAAAABAF54T6WWGG33NNVSW45C7OR4XAZNKIRUXGY3VONZWS33OVJRW63LNMVXHIX3JMTHAAWS7QI.gif"
          alt="" moz-do-not-send="true" width="1" height="1"><span
style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message
          ID: <span>&lt;PRIME-RE/prime-re</span><span>.</span><span>github</span><span>.</span><span>io/repo-discussions/82</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>
      <script type="application/ld+json">[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/PRIME-RE/prime-re.github.io/discussions/82",
"url": "https://github.com/PRIME-RE/prime-re.github.io/discussions/82",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]</script>
      -- <br>
      You received this message because you are subscribed to the Google
      Groups "NHP-MRI" group.<br>
      To unsubscribe from this group and stop receiving emails from it,
      send an email to <a
        ***@***.***"
        moz-do-not-send="true" ***@***.***</a>.<br>
      To view this discussion on the web visit <a
href="https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/82%40github.com?utm_medium=email&amp;utm_source=footer"
        moz-do-not-send="true">https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/82%40github.com</a>.<br>
    </blockquote>
    <pre class="moz-signature" cols="72">
</pre>
  </body>
</html>

--------------u4ST6ARkbHSedjefODSYnNVb--

**Reply by SjoerdMurris** [reply]
Hi Igor,

Thanks for your suggestion! I went through your paper before, which I think is great both in terms of both content and visualization.

We do have some inflated surfaces of NMT generated through freesurfer tools (I'm a cheapskate when it comes to software packages), but I'm specifically looking for flatmaps. What do you think about the NeuroElf package in MATLAB, is it easy to use?
Leaning towards using freesurfer tools for reconstruction and python-based tools (PyCortex) for visualization purposes.

Cheers,
Sjoerd

**Comment by PRIME-DRE** [comment]
Dear Sjoerd,

Thanks, and apologies for delayed response! No, I don't think NeuroElf can
do flatmaps - but yes, it is easy to use (object-oriented code) and easy to
integrate into other pipelines.

Cheers,
Igor
